dev/bots/utils.dart

```diff
@@ -343,8 +342,8 @@ Future<void> runDartTest(String workingDirectory, {
   bool collectMetrics = false,
 }) async {
   int? cpus;
-  final String? cpuVariable = Platform.environment['CPU']; // CPU is set in cirrus.yml
-  if (cpuVariable != null) {
+  // CPU is set in cirrus.yml
+  if (Platform.environment case {'CPU': final String cpuVariable}) {
```

Comment:
This is not related to this PR but having the word variable in a variable is, well, redundant.

Reply:
yes absolutely 💯

…or maybe it _is_ related to this PR: one nice thing about if-case is that the matched variable is scoped to the if-statement's body, so a concise name can be used without overlapping something else in the same function.